app/controllers/message_objects_controller.rb

```diff
@@ -49,7 +49,7 @@ def signing_data
     authorize @message_object
 
     head :no_content and return unless @message_object.content.present?
-    render template: 'message_drafts/update_body' and return unless @message.valid?(:validate_data)
+    render template: 'message_drafts/update_body' and return if @message_object.form? && !@message.valid?(:validate_data)
```

Comment:
rozdelme to na dva riadky lebo su to rozne podmienky

```suggestion
    return unless @message_object.form?
    render template: 'message_drafts/update_body' unless @message.valid?(:validate_data)
```
?

Reply:
Upravene